src/kemal/exception_handler.cr

```diff
@@ -11,12 +11,26 @@ module Kemal
     rescue ex : Kemal::Exceptions::CustomException
       call_exception_with_status_code(context, ex, context.response.status_code)
     rescue ex : Exception
+      # Use error handler defined for the current exception if it exists
+      return call_exception_with_exception(context, ex, 500) if Kemal.config.error_handlers.has_key?(ex.class)
```

Comment:
This line is added above the `log`  function in order for the specs to pass.

A logger is needed for `#log` to be able to successfully run  which the tests in `exception_handler_spec.cr` doesn't appear to have defined. 

This shouldn't be a problem in the real world case as a logger would always exist afaik.

Please let me know if this should be rectified in some other way.